core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -1543,6 +1547,76 @@ class PlaintextAdminIntegrationTest extends BaseAdminIntegrationTest {
     assertNull(returnedOffsets.get(topicPartition))
   }
 
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
+  @MethodSource(Array("getTestQuorumAndGroupProtocolParametersClassicGroupProtocolOnly"))
```

Comment:
I didn't use `getTestQuorumAndGroupProtocolParametersAll` here as we found that under async consumer, it wont throw exception in [L1598](https://github.com/apache/kafka/pull/17840/files#diff-41e66d9de25929a78b95f153b1ea93476851a379085d4eb7a6d118459f711b1aR1598)
```java
    assertEquals("Encountered corrupt message when fetching offset 0 for topic-partition topic-0",
      assertThrows(classOf[KafkaException], () => consumer.poll(JDuration.ofMillis(DEFAULT_MAX_WAIT_MS))).getMessage)
```